libraries/AP_HAL_ChibiOS/system.cpp

```diff
@@ -203,6 +204,9 @@ void panic(const char *errormsg, ...)
     vprintf(errormsg, ap);
     va_end(ap);
 
+    gcs().send_text(MAV_SEVERITY_ALERT, errormsg, ap);
```

Comment:
This is going to cross a lot of code - some of which might be causing the panic in the first place!  The way this is currently structured would stop us sending the message to the console, which would be bad.

We'd want some sort of recursion protection in place to avoid that - and now we're getting complicated.